packages/web/components/Landing/OnboardingGame/Chiev.tsx

```diff
@@ -0,0 +1,431 @@
+/* eslint-disable no-nested-ternary */
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Text,
+  Tooltip,
+  usePrefersReducedMotion,
+  VStack,
+} from '@metafam/ds';
+import { httpLink } from '@metafam/utils';
+import { animated, useSpring } from '@react-spring/web';
+import OctoBg from 'assets/baby_octo.png';
+import { LandingConnectButton } from 'components/Landing/LandingConnectButton';
+import { MetaLink } from 'components/Link';
+import { CONFIG } from 'config';
+import { useGame } from 'contexts/GameContext';
+import { Contract, providers } from 'ethers';
+import { useWeb3 } from 'lib/hooks';
+import { get } from 'lib/store';
+import {
+  Dispatch,
+  SetStateAction,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { shortenAddress } from 'utils/stringHelpers';
+
+import ABI from '../../../contracts/BulkDisbursableNFTs.abi';
+
+export interface IChievMetadata {
+  name: string;
+  description: string;
+  image: string;
+  animation_url?: string;
+}
+
+export const Chiev = ({
+  won,
+  setWon,
+}: {
+  won: boolean;
+  setWon: Dispatch<SetStateAction<boolean>>;
+}) => {
+  const prefersReducedMotion = usePrefersReducedMotion();
+  const [noMotion, setNoMotion] = useState(false);
+  const root = typeof window !== 'undefined' ? document.body : null;
+  const { mintChiev, txLoading } = useGame();
+  const [claiming, setClaiming] = useState(false);
+  const { address: account, chainId, connected } = useWeb3();
+  const [wrongNetwork, setWrongNetwork] = useState(false);
+  const { alchemyApiKey, chievContractAddress: contractAddress } = CONFIG;
+  const claimed = get('ChievClaimed') === 'true' ?? false;
+  const chievId = BigInt(
```

Comment:
Lets capitalize constants as a naming convention. 
And lets move them outside the component.

Reply:
> I gave it a quick look and left a bunch of comments.

Amazing! Thank you for the time taken to check this over @dan13ram. I've responded or emoji'd all comments and will address them over the next couple of days.  

> Overall this is just amazing!!! Fantastic work @luxumbra pray

I appreciate that, thanks. 😊 

> I'll take another look once you fix these comments :)

I appreciate you. 🐙